tests/integration_restrict.sh

```
@@ -26,18 +26,45 @@ chk() { # desc expected actual
+# reports 'reject' and fails CI). Publishing an acceptable throwaway event until
+# it succeeds converts "port open" into "pipeline ready", and is independent of
+# noz's error-output format so it needs no reject-vs-timeout parsing.
+warmup() {
+  local args
+  case "$MODE" in
+    auth)      args=(--sec "$SEC1" --auth -c warmup) ;;
+    protected) args=(--sec "$SEC1" -c warmup) ;;
+    pow)       args=(--sec "$SEC1" --pow 8 -c warmup) ;;
+    *)         return 0 ;; # unknown mode is reported by the assertion case below
+  esac
+  for i in $(seq 1 40); do
```

Shellcheck (0.11.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)